K-mer.scala code

K-mer.scala code given at https://github.com/bigdatagenomics/adam is giving me errors.
I ran that code in two ways "

> adam-shell -i kmer.scala

> scalac kmer.scala

I am getting following errors
$scalac kmer.scala
kmer.scala:5: error: expected class or object definition
val ac = new ADAMContext(sc)
^
kmer.scala:7: error: expected class or object definition
val reads = ac.loadAlignments(
^
kmer.scala:19: error: expected class or object definition
val kmers =
^
kmer.scala:26: error: expected class or object definition
kmers.take(10).foreach(println)
^
four errors found


And when i run on spark "adam-shell -i kmer.scala". I am getting following errors
<console>:19: error: object rdd is not a member of package org.bdgenomics.adam
       import org.bdgenomics.adam.rdd.ADAMContext
                                  ^
<console>:19: error: object rdd is not a member of package org.bdgenomics.adam
       import org.bdgenomics.adam.rdd.ADAMContext._
                                  ^
<console>:19: error: object projections is not a member of package org.bdgenomics.adam
       import org.bdgenomics.adam.projections.{ AlignmentRecordField, Projection }
                                  ^
<console>:21: error: not found: type ADAMContext
       val ac = new ADAMContext(sc)
                    ^
<console>:19: error: not found: value ac
       val reads = ac.loadAlignments(
                   ^
<console>:20: error: not found: value reads
         reads
         ^
<console>:1: error: illegal start of definition
           .flatMap(_.getSequence.sliding(21).map(k => (k, 1L)))
           ^
<console>:1: error: illegal start of definition
           .reduceByKey(_ + _)
           ^
<console>:1: error: illegal start of definition
           .map(_.swap)
           ^
<console>:1: error: illegal start of definition
           .sortByKey(ascending = false)
           ^
<console>:20: error: not found: value kmers
              kmers.take(10).foreach(println)
              ^
Kindly help me to solve the issues.

Regards
Pavan

--- a/bin/adam-shell
+++ b/bin/adam-shell
@@ -31,17 +31,17 @@ ADAM_CLI_JAR=
 if [ -d "$SCRIPT_DIR/repo" ]; then
   ASSEMBLY_DIR="$SCRIPT_DIR/repo"
 else
-  ASSEMBLY_DIR="$SCRIPT_DIR/adam-cli/target"
+  ASSEMBLY_DIR="$SCRIPT_DIR/adam-assembly/target"
 fi
 
-num_jars="$(ls -1 "$ASSEMBLY_DIR" | grep "^adam-cli_[0-9A-Za-z\.-]*\.jar$" | grep -v javadoc | wc -l)"
+num_jars="$(ls -1 "$ASSEMBLY_DIR" | grep "^adam_[0-9A-Za-z\.-]*\.jar$" | grep -v javadoc | wc -l)"
 if [ "$num_jars" -eq "0" ]; then
-  echo "Failed to find ADAM cli assembly in $ASSEMBLY_DIR." 1>&2
+  echo "Failed to find ADAM assembly in $ASSEMBLY_DIR." 1>&2
   echo "You need to build ADAM before running this program." 1>&2
   exit 1
 fi
 
-ASSEMBLY_JARS="$(ls -1 "$ASSEMBLY_DIR" | grep "^adam-cli_[0-9A-Za-z\.-]*\.jar$" | grep -v javadoc || true)"
+ASSEMBLY_JARS="$(ls -1 "$ASSEMBLY_DIR" | grep "^adam_[0-9A-Za-z\.-]*\.jar$" | grep -v javadoc || true)"
 if [ "$num_jars" -gt "1" ]; then
   echo "Found multiple ADAM cli assembly jars in $ASSEMBLY_DIR:" 1>&2
   echo "$ASSEMBLY_JARS" 1>&2

